packages/studio-plugin/src/sourcefiles/ComponentFile.ts

```diff
@@ -62,8 +63,9 @@ export default class ComponentFile {
 function getCssFilesFromDependencyTree(dependencyTree: Tree): string[] {
   const cssFiles = Object.entries(dependencyTree).reduce(
     (cssFiles, [filename, subDependencyTree]) => {
-      if (filename.includes(".css")) {
-        cssFiles.add(filename);
+      const unixFilepath = upath.toUnix(filename);
```

Comment:
can this conversion go inside the if-statement below since it's only needed if the file is a CSS file?

Reply:
done!